README.md

```diff
@@ -1 +1,46 @@
-# Portflio-Mobile-First
\ No newline at end of file
+# Portfolio Web Page
+
+> Built a simple Html and css project 
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
```

Comment:
- [ ] Please make your readme descriptive so that the recruiter or anyone wants to use the project can follow through easily.  please follow the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) and make amends.